assets/locales/en.json

```diff
@@ -141,5 +135,27 @@
   "runGarbageCollectorDone": {
     "title": "Garbage collector",
     "message": "The garbage collector ran successfully."
+  },
+  "downloadCidContentDialog": {
+    "title": "Download",
```

Comment:
```suggestion
    "title": "Download from CID",
```

Reply:
As per the comment above, I'll be keeping this Download.